build-system/compile/bundles.config.extensions.json

```
@@ -199,6 +199,12 @@
   },
   {"name": "amp-facebook-like", "version": "0.1", "latestVersion": "0.1"},
   {"name": "amp-facebook-page", "version": "0.1", "latestVersion": "0.1"},
+  {
+    "name": "amp-facebook-page",
+    "version": "1.0",
+    "options": {"hasCss": true},
+    "latestVersion": "0.1"
```

nit: `latestVersion` after `version`. I'll change the generator so that it does the right thing.

```suggestion
    "version": "1.0",
    "latestVersion": "0.1"
    "options": {"hasCss": true},
```

---

#34713 for generator change

---

Done, thanks